features/step_definitions/all_my_step_definitions_step.rb

```diff
@@ -0,0 +1,25 @@
+Given("the following user exists") do |table|
+    table.hashes.each do |user|
+      User.create(user)
+end
+  end
+  
+  Given("I am on the landing page") do
+    visit '/'
+  end
+  
+  When("I click on the link {string}") do |login|
+    click_on(login)
+  end
+  
+  When("I fill in the {string} with {string}") do |a, b|
```

Comment:
You got to be kidding me, right? Single letter variable names are a TOTAL no-go! This IS a blocker. 😡